### PR TITLE
Resolves #10

### LINKED ISSUE #10: Inconsistent times displayed on View Seminar page
On "View Seminar" pages like https://mathseminars.org/seminar/ComplexGeometry, the time for the seminar is based on the time zone of the seminar (which is not displayed) while the time zone of the talk is based on the viewer's time zone.  This is very confusing (so confusing that I consider it a bug, a person looking at this page has no idea what  time the talks are).

I think the best thing to do would be to display the time zone of the seminar and display the time in both the seminar and viewer time zones (e.g. captioned "Seminar time" and "Your time").  We should also indicate that the talks are in the viewer's time zone, perhaps by changing the date/time header to "Your time".

### PR DESCRIPTION
**Pull request by AndrewVSutherland**
On View seminar and View talk pages shows the meeting time both in the users time zone and the seminar time zone if they are different.  This can happen when the times are actually the same (e.g. US/Eastern and America/New York), which is slightly annoying but at least not confusing.

### CODE REVIEW

**Comment by AndrewVSutherland** [comment]
Changed to only show two times when the seminar or talk is not online (so to test you may need to briefly switch a seminar or talk to offline).

File: seminars/templates/talk.html [hunk 1]
```diff
@@ -26,6 +26,10 @@ <h1>{{talk.show_title()}}{% if not talk.display %} (hidden){% elif talk.online %
   <p>
     {{ talk.show_time_and_duration() | safe }}{% if talk.timezone != current_user.timezone %} in your time zone, {{ current_user.timezone.replace("_", " ") }}.{% endif %}
   </p>
+  {% if not talk.online and talk.timezone != current_user.timezone %}
```
Comment (roed314) [inline seminars/templates/talk.html:29]: Is it better for this to be "not talk.online" or "talk.room"?  I'm fine with the way you've written it, but I think the plan is for the presence of a room to indicate an offline presence, which is maybe what you're looking for.

**Comment by AndrewVSutherland** [comment]
@roed314 tz is used in the code, see the new line 169 of talk.py.  But I just changed it to a property rather than an attribute, to match seminar.py.

I think I like (not talk.online or talk.room) best.  I made that change.

File: seminars/talk.py [hunk 2]
```diff
@@ -53,6 +53,7 @@ def __init__(
             self.token = "%016x" % random.randrange(16 ** 16)
             self.display = seminar.display
             self.online = getattr(seminar, "online", bool(seminar.live_link))
+            self.tz = seminar.tz
```
Comment (roed314) [inline seminars/talk.py:56]: Properties don't work this way (Sage `lazy_attributes` do).  If you make `tz` a property, you need a setter in order to assign to it.  Maybe set `self.timezone = seminar.timezone` instead?

File: seminars/talk.py [hunk 3]
```diff
@@ -123,6 +124,10 @@ def editable_end_time(self):
         """
         return self._editable_time(self.end_time)
 
+    @property
+    def tz(self):
+        return pytz.timezone(self.timezone)
```
Comment (roed314) [inline seminars/talk.py:129]: We might want to protect against `timezone` being invalid (e.g. `None`), though I think we don't allow that.

File: seminars/talk.py (hunk 2 above)
Comment (AndrewVSutherland) [reply seminars/talk.py:56]: Sorry, I just forgot to change that line, fixed.

File: seminars/talk.py (hunk 3 above)
Comment (AndrewVSutherland) [reply seminars/talk.py:129]: We don't do this in seminar,py, so I don't think we need to (but if we do we should do it in both places.

**Pull request merged**